roboflow/models/classification.py

```diff
@@ -13,7 +13,16 @@
 
 
 class ClassificationModel:
-    def __init__(self, api_key, id, name=None, version=None, local=False):
+    def __init__(
+        self,
+        api_key,
+        id,
+        name=None,
+        version=None,
+        local=False,
+        colors={},
+        preprocessing={},
+    ):
```

Comment:
This is wrong, you shouldn't use mutable classes as parameters, set them to `None` and create it if is `None`

`colors=None`

Then in the `__init__`

`self.colors = {} if colors is None else colors`

Reply:
Ok, please level me up @FrancescoSaverioZuppichini, why is that?